validator/lib/validateExtensions.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+const Cesium = require('cesium');
+
+const defaultValue = Cesium.defaultValue;
+const defined = Cesium.defined;
+
+module.exports = validateExtensions;
+
+const EMPTY_ARRAY = [];
+
+const requiredExtensions = ['3DTILES_content_gltf'];
+
+/**
+ * Check if a tileset's extensions are valid.
+ *
+ * @param {Object} options An object with the following properties:
+ * @param {Object} options.tileset The tileset JSON.
+ * @returns {String} An error message if validation fails, otherwise undefined.
+ */
+function validateExtensions(options) {
+    const tileset = options.tileset;
+    const extensions = tileset.extensions;
+    const extensionsUsed = defaultValue(tileset.extensionsUsed, EMPTY_ARRAY);
+    const extensionsRequired = defaultValue(tileset.extensionsRequired, EMPTY_ARRAY);
+
+    if (defined(extensions)) {
+        for (const extensionName in extensions) {
+            if (extensions.hasOwnProperty(extensionName)) {
+                const extension = extensions[extensionName];
+                if (!extensionsUsed.includes(extensionName)) {
```

Comment:
We need to check if `extensionsUsed` and `extensionsRequired` are array before we call `.includes()`.

Reply:
Fixed